src/main/java/com/googlesource/gerrit/plugins/oauth/KeycloakApi.java

```diff
@@ -22,7 +22,7 @@
 
 public class KeycloakApi extends DefaultApi20 {
 
-  private static final String AUTHORIZE_URL = "%s/auth/realms/%s/protocol/openid-connect/auth";
+  private static final String AUTHORIZE_URL = "%s/realms/%s/protocol/openid-connect/auth";
```

Comment:
This would require a change in the doc to append /auth to the config root url depending on your keycloak version